Crud_Blazor/Pages/Index.razor

```diff
@@ -1,5 +1,72 @@
 ﻿@page "/"
+@using Crud_Blazor.Components
+@using Crud_Blazor.Data
+@inject IJSRuntime _JsRuntime
+@inject ServiceComponent Service
 
-<h1>Hello, world!</h1>
+<div class="col-md-6">
 
-<p>Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum.</p>
+    User ID:
+    <input class="form-control" @bind="userName" />
+    PassWord:
+    <input class="form-control" @bind="PassWord" />
+    <br />
+
+    <button class="btn btn-info" @onclick="@(()=>CreateToken())">Login/Create Token</button>
+    <button class="btn btn-info" @onclick="@(()=>PostMethod())">Post</button>
+
+    <hr />
+    <br />
+    <label class="col-form-label">@Bearer</label>
+    <label class="col-form-label">@Message</label>
+</div>
+@code{
+    string userName;
+    string PassWord;
+    string Message;
+
+    [Parameter]
+    public string Bearer { get; set;  }
+
+    public void CreateToken()
+    {
+        Message = "";
+        Bearer = "";
+
+        var response = Service.ResponseJson("http://localhost:36915/api/Login?userId=" + userName + "&pass=" + PassWord + "", null, null, null,RestSharp.Method.GET);
```

Comment:
use base url at top and add variable
var baseURL = "http://localhost:36915/api/";
var response = Service.ResponseJson(baseURL + "Login?userId=" + userName + "&pass=" + PassWord + "", null, null, null,RestSharp.Method.GET);